bundles/binding/org.openhab.binding.zwave/database/linear/wadwaz-1.xml

```diff
@@ -22,41 +22,8 @@
 			<id>0xFF</id>
 		</Class>
 	</CommandClasses>
-	<Configuration>
-		<Parameter>
-			<Index>1</Index>
-			<Type>short</Type>
-			<Default>0</Default>
-			<Size>2</Size>
-			<Label Lang="en">IN1 Alarm Cancellation Delay alarm Cancellation
-			</Label>
-			<Help Lang="en">Additional delay after an alarm from input IN1 has
-				ceased. The parameter allows you to specify additional time, after
-				which the input no. 1 alarm is cancelled once its violation has
-				ceased.
-			</Help>
-		</Parameter>
-		<Parameter>
-			<Index>2</Index>
-			<Type>byte</Type>
-			<Default>1</Default>
-			<Size>1</Size>
-			<Label Lang="en">Status change signaled by LEDs change signaled by
-			</Label>
-		</Parameter>
-	</Configuration>
 	<Associations>
 		<Group>
-			<Index>1</Index>
-			<Maximum>5</Maximum>
-			<Label lang="en">Input IN1</Label>
-		</Group>
-		<Group>
-			<Index>2</Index>
-			<Maximum>5</Maximum>
-			<Label lang="en">Tamper Button</Label>
-		</Group>
-		<Group>
 			<Index>3</Index>
```

Comment:
After going through the Ide setup I have 14 files showing as changed in
github window. Can I just click ignore on them to get them to go away
before I accidentally commit one of them?
On Jun 27, 2015 3:04 PM, "Hydrilla Gorilla" <hydrillagorilla@gmail.com>
wrote:

> Lol i wondered about that group number checking manual and  changing!
> On Jun 27, 2015 3:02 PM, "Chris Jackson" <notifications@github.com> wrote:
>
>> In bundles/binding/org.openhab.binding.zwave/database/linear/wadwaz-1.xml
>> <https://github.com/openhab/openhab/pull/2813#discussion_r33415651>:
>>
>> >  	<Associations>
>> >  		<Group>
>> > -			<Index>1</Index>
>> > -			<Maximum>5</Maximum>
>> > -			<Label lang="en">Input IN1</Label>
>> > -		</Group>
>> > -		<Group>
>> > -			<Index>2</Index>
>> > -			<Maximum>5</Maximum>
>> > -			<Label lang="en">Tamper Button</Label>
>> > -		</Group>
>> > -		<Group>
>> >  			<Index>3</Index>
>>
>> I think the index of the remaining association group should be 1
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/openhab/openhab/pull/2813/files#r33415651>.
>>
>